birdhouse/config/jupyterhub/jupyterhub_config.py.template

```diff
@@ -49,9 +66,10 @@ if len(host_gdrive_settings_path) > 0:
         "mode": "ro"
     }
 
+#  Mount folder containing the notebooks that should be available to all images
 host_tutorial_notebooks_dir = join(jupyterhub_data_dir, "tutorial-notebooks")
-c.DockerSpawner.volumes[host_tutorial_notebooks_dir] = {
-    "bind": join(notebook_dir, "tutorial-notebooks"),
+c.DockerSpawner.volumes[join(host_tutorial_notebooks_dir, "common")] = {
+    "bind": join(notebook_dir, "tutorial-notebooks/common"),
```

Comment:
The problem was that if we leave notebooks directly in the tutorial-notebooks folder and mounting the tutorial-notebooks folder to include those notebooks, it will mount the specific image notebooks folder at the same time since they are too in the tutorial-notebooks.
We could find another way to structure it, but we have to make sure there is no resulting conflicts, and that the different folders exclude each other.

Reply:
In fact, common means exactly that, the common things, core to pavics, like thredds, magpie, geoserver, eventually weaver as the entry point for every wps (but generic), the catalog. Raven, Finch and the climate birds should be moved in a climate image.

Reply:
For the sake of seemless backward-compat, I suggest moving the volume-mount of this `common` dir inside the `if(os.environ['MOUNT_IMAGE_SPECIFIC_NOTEBOOKS'] == 'true')` and add an `else` clause to restore the volume-mount of just the `tutorial-notebooks` folder as if this change was never there.

So if MOUNT_IMAGE_SPECIFIC_NOTEBOOKS'] == 'true', Jupyter user will see
```
/tutorial-notebooks/common/  --> /data/jupyterhub_user_data/tutorial-notebooks/common/
                    crim-eo/  --> /data/jupyterhub_user_data/tutorial-notebooks/crim-eo/
```
else
```
/tutorial-notebooks/ --> /data/jupyterhub_user_data/tutorial-notebooks/
```